client/components/course/Settings/UserManagement/UserList.vue

```diff
@@ -8,9 +8,7 @@
     <template v-slot:items="{ item }">
       <td class="text-xs-left">
         <v-avatar color="blue lighten-1" size="40" dark class="mr-3">
-          <span class="headline white--text">
-            {{ item.email[0].toUpperCase() }}
-          </span>
+          <img :src="item.imgUrl"/>
```

Comment:
Maybe adding `alt` attribute to image tag could be a good idea:
```vue
  <img :src="item.imgUrl"/ :alt="item | username">
``` 
`username (or fullname)` can be filter that can return users full name. 
Also what's the case when user doesn't have a profile image? Maybe using the `v-avatar` [functionality](https://vuetifyjs.com/en/components/avatars#icon-and-text) to generate the avatar from first letter of the user name. 
```vue
<v-avatar color="blue lighten-1" size="40" dark class="mr-3">
  <img  v-if="item.imgUrl" :src="item.imgUrl"/ :alt="item | username ">
  <span v-else class="white--text headline">J</span>
</v-avatar>
```

Reply:
For `alt` attribute :+1:
Other, explained in comment :point_down:

Reply:
👍